tests/test_lib_tokens_webauthn.py

```diff
@@ -274,7 +274,9 @@ def test_03_token_update(self):
         self.assertEqual(PUB_KEY, self.token.get_tokeninfo(WEBAUTHNINFO.PUB_KEY))
 
     def test_04_authentication(self):
-        web_authn_sign_request = self._create_challenge().get("webAuthnSignRequest")
+        reply_dict = self._create_challenge()
+        attributes = reply_dict.get("attributes")
+        web_authn_sign_request = attributes.get("webAuthnSignRequest")
```

Comment:
just to be sure we should check that the ``client_mode`` here is ``webauthn``

Reply:
The client mode is not added by the ``tokenclass.create_challenge`` but centrally in ``lib/token.py::check_token_list`` aka ``create_challenges_from_tokens``.
See: https://github.com/privacyidea/privacyidea/pull/2782/files#diff-47e7d7bbee3d5a5017148691cf20ddaa8fc5881be6493176e2206e980bb6caa6R2098

So at this place we are testing the token lib level, where there is not client mode.